### ISSUE TITLE
Ship gone and inventory cleared Regalis->Boletarian

### ISSUE THREAD

**Issue by Gaming4me** [StarQuestMinecraft/StarQuestPublic#364]
So me and McDoubleAwesome were doing a spawn run Val-> Spawn, flying at minimum height and invisible. When we entered bole from below, south/southeast-wise while autopiloting by accident, we were flying a maxed out heavy freighter filled with pumpkins. both of our ships and inventories were gone, we ended up dying but we did not get warped to our cryos, instead we both respawned on bole on the same place. We were assisted by oldskoolish, who has more details on location and time.

Also this happened about 30 mins ago.

**Comment by Gaming4me** [comment]
Alright, as me and McDouble were leaving Bole with the restored ships and inventories, as the ship reached max height, it attempted to leave bole, I got the usual messages until the [ALERT] leaving the atmosphere message, but instead of leaving bole, we were just sitting there on the ship, which was now unpilotable and unmovable for about 15 seconds until bole crashed.

**Comment by mcdouble** [comment]
Yes almost immediately after oldskool left the ship to leave us on our merry way, we flew for MAYBE 2 seconds before I got invwiped and gaming couldn't fly the ship anymore. Average chat confusion happened, then crash.

**Comment by Gaming4me** [comment]
oh and as it turns out, the ships droppers were also wiped. Once bole cabe back online I relogged and flew the ship to spawn (did not check the droppers for I was in a hurry for a job meeting). I dropped the ship at spawn and now I realized the dispensers were empty. Since there was the possibility of someone having glitched through the lockette door, I asked liggo to reload the ship so we could see if the pumpkins were still there when I flew it bole-spawn. As it turns out the ships droppers were already empty from the moment bole came back online.
